src/Components/getList.js

```diff
@@ -11,11 +24,25 @@ class GetList extends React.Component {
           return isLoading ? (
             <div>loading...</div>
           ) : (
-            <div>
+            <div className="grocery-list">
               <h1>Groceries</h1>
-              <ul>
+              <ul className="tl f2">
                 {data.map(item => (
-                  <li key={item.id}>{item.itemName}</li>
+                  <li className="bg-white pa1 shadow" key={item.id}>
+                    <label>
+                    <input 
+                      type="checkbox"
+                      ischecked={this.state.toggleCheck}
+                      defaultChecked={
+                        item.lastPurchasedDate
+                          ? Date.now() / 1000 - item.lastPurchasedDate.seconds <
```

Comment:
I want to tell you how impressed I am that you solved this problem in this way! Computing if the checkbox is checked by calculating it on-the-fly here is great! I think way is the best way to solve this because you only need to store one piece of state on the item, and can compute the rest from that. 🎉 👏